pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -0,0 +1,91 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package virtwrap
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/network/vmispec"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+var _ = Describe("nic hotplug on virt-launcher", func() {
+	const (
+		nadName     = "n1n"
+		networkName = "n1"
+	)
+
+	DescribeTable("networksToHotplugWhoseInterfacesAreNotInTheDomain", func(vmi *v1.VirtualMachineInstance, domainIfaces map[string]api.Interface, expectedNetworks []v1.Network) {
+		Expect(
+			networksToHotplugWhoseInterfacesAreNotInTheDomain(vmi, domainIfaces),
+		).To(ConsistOf(expectedNetworks))
+	},
+		Entry("vmi with no networks, and no interfaces in the domain",
+			&v1.VirtualMachineInstance{Spec: v1.VirtualMachineInstanceSpec{Networks: []v1.Network{}}},
+			map[string]api.Interface{},
+			nil,
+		),
+		Entry("vmi with 1 network, and an associated interface in the domain",
+			&v1.VirtualMachineInstance{
+				Spec: v1.VirtualMachineInstanceSpec{Networks: []v1.Network{generateNetwork(networkName, nadName)}},
+			},
+			map[string]api.Interface{networkName: {Alias: api.NewUserDefinedAlias(networkName)}},
+			nil,
+		),
+		Entry("vmi with 1 network (whose PodConfig is *not* done), but no interfaces in the domain",
+			&v1.VirtualMachineInstance{
+				Spec: v1.VirtualMachineInstanceSpec{Networks: []v1.Network{generateNetwork(networkName, nadName)}},
+			},
+			map[string]api.Interface{},
+			nil,
+		),
+		Entry("vmi with 1 network (with PodConfigDone: true), but already present in the domain",
```

Comment:
1. Same, please update the description, PodConfigDone doesn't exist anymore.
2. I think the test duplicates "vmi with 1 network, and an associated interface in the domain".

Reply:
Done, good catch.